textworld/generator/game.py

```diff
@@ -1000,31 +1008,46 @@ def __init__(self):
         self._kb = None
         self._seeds = None
 
+        self.nb_parallel_quests = 1
         self.nb_rooms = 1
         self.nb_objects = 1
-        self.quest_length = 1
-        self.quest_breadth = 1
         self.force_recompile = False
         self.file_ext = ".ulx"
         self.path = "./tw_games/"
 
     @property
     def quest_length(self) -> int:
-        return self.chaining.max_depth
+        assert self.chaining.min_length == self.chaining.max_length
+        return self.chaining.min_length
 
     @quest_length.setter
     def quest_length(self, value: int) -> None:
-        self.chaining.min_depth = 1
+        self.chaining.min_length = value
+        self.chaining.max_length = value
         self.chaining.max_depth = value
+        # self.chaining.max_breadth = value
+
+    # @property
+    # def quest_depth(self) -> int:
```

Comment:
I need to delete those commented properties, I don't think there is a use for setting the chain's depth this way (i.e. better to directly change the `chaining.min/max_depth`).